internal/tss/utils/transaction_service.go

```diff
@@ -0,0 +1,240 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/stellar/go/clients/horizonclient"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/wallet-backend/internal/signing"
+	"github.com/stellar/wallet-backend/internal/tss"
+	tssErr "github.com/stellar/wallet-backend/internal/tss/errors"
+)
+
+var (
+	RpcPost                 = http.Post
+	UnMarshalRPCResponse    = io.ReadAll
+	UnMarshalJSON           = parseJSONBody
+	callRPC                 = sendRPCRequest
+	UnMarshalErrorResultXdr = parseErrorResultXdr
+)
+
+type transactionService struct {
```

Comment:
Let's add an `var _ TransactionService = (*transactionService)(nil)` to make sure the struct implements the interface.

Reply:
Done